docs/pages/blog/2022-tenerife-retreat.md

```diff
@@ -0,0 +1,190 @@
+---
+title: MUI's company retreat in Tenerife: a recap
+description: Our internationally distributed startup gathered on a remote island to get to know each other better. Here's what happened!
+date: 2022-07-28T00:00:00.000Z
+authors: ['samuelsycamore']
+card: false
+tags: ['Company']
+---
+
+One of the toughest challenges to overcome as a fully remote team is fostering a supportive and inclusive company culture.
+How do you build trust, comradery, and—dare I say it?—_friendships_, when there are so few opportunities for spontaneous social interactions in our daily work routines?
+
+At MUI, we've chosen to prioritize in-person company retreats as a way to grow together and cultivate the culture that informs how we operate.
+
+![Photo of the MUI team in front of the pool at our accommodations in Tenerife](/static/blog/2022-tenerife-retreat/group-photo.jpeg)
+
+<p class="blog-description">The MUI team poses next to the pool during the company retreat.</p>
+
+In late June 2022, sixteen of us gathered at a remote private residence on the island of Tenerife.
+This was MUI's second company retreat, the first being in Portugal in 2021 when the team was just a fraction of the size it is today.
+
+We had a ton of fun getting to know each other better over oceanic activities, poolside conversations, and incredible catered meals—not to mention a steady supply of Spanish wine. 🥂
```

Comment:
```suggestion
We had a ton of fun getting to know each other better over oceanic activities, poolside conversations, and incredible catered meals—not to mention a steady supply of local wine. 🥂
```
Not sure what we had at the restaurants, but the wine I bought for the last day/night was all from Tenerife. (Which of course makes it Spanish, but local sounds better. 😄 )